src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -376,7 +377,7 @@ export default class AirportModel {
         this.setActiveRunwaysFromNames(data.arrivalRunway, data.departureRunway);
         this.buildAirportMaps(data.maps);
         this.buildRestrictedAreas(data.restricted);
-        this.updateCurrentWind(data.wind);
+        AirportWind.setStaticWind(data.wind);
```

Comment:
I should have warned you that doing it this way, though easier, adds a little more complexity to getting things connected.  Since we are now using a class for `AirportWind`, you will need to use the `this.wind` property to hold this instance:
```javascript
this.wind = new AirportWindModel.(data.wind);
```

_(sorry, I'm changing my mind on the name again.  I think `AirportWindModel` is better)_